### ISSUE TITLE
Function arity error on :input when using Stream.resource/3

### ISSUE THREAD

**Issue by bfolkens** [akash-akya/ex_cmd#18]
Passing a `Stream.resource/3` (or other simple stream items) results in the following error:

```elixir
** (ArgumentError) :input must be either Enumerable or a function with arity 1
    (ex_cmd 0.8.0) lib/ex_cmd/stream.ex:65: ExCmd.Stream.start_input_streamer/2
    (ex_cmd 0.8.0) lib/ex_cmd/stream.ex:44: ExCmd.Stream.__build__/2
    iex:4: (file)
```

**Comment by akash-akya** [comment]
Thanks for the issue. 
Yes, this is an edge case. I noticed this recently for other lib and [did fix](https://github.com/akash-akya/exile/pull/17/) it there.

**Comment by akash-akya** [comment]
Fixed in #20
